Podfile

```diff
@@ -1,16 +1,16 @@
 # Uncomment this line to define a global platform for your project
 platform :ios, '8.0'
 
-target 'FirebaseUI' do
-  use_frameworks!
+use_frameworks!
 
-  pod 'Firebase'
-  pod 'Firebase/Database'
-  pod 'Firebase/Auth'
+target 'FirebaseUI' do
+  # pod 'Firebase'
```

Comment:
We can just delete these if they're being included in the individual ones. Curious what FirebaseUI does (it seems like it's just a container now).